easybuild/easyconfigs/f/funannotate/funannotate-1.8.17-foss-2023a.eb

```diff
@@ -77,10 +77,6 @@ sanity_check_paths = {
     'dirs': [],
 }
 
-modextrapaths = {
-    'GENEMARK_PATH': '$EBROOTGENEMARKMINET',
```

Comment:
This may no longer be necessary once https://github.com/easybuilders/easybuild-framework/pull/4855 is merged?

Reply:
yes i'll do https://github.com/easybuilders/easybuild-easyconfigs/pull/22851 instead